components/script/dom/console.rs

```diff
@@ -65,13 +73,80 @@ fn console_message(global: &GlobalScope, message: DOMString, level: LogLevel) {
     })
 }
 
+//generate matrix of table elements
```

Comment:
Documentation of functions in rust starts with three "/" and should be written with complete sentences. This allows the documentation to show up in IDEs and also be generated properly on the documentation site.